vorestation.dme

```diff
@@ -1008,12 +1008,14 @@
 #include "code\game\objects\items\contraband.dm"
 #include "code\game\objects\items\contraband_vr.dm"
 #include "code\game\objects\items\crayons.dm"
+#include "code\game\objects\items\falling_object.dm"

```

Comment:
I would like the filename to have _vr on it, mostly because it would make it a bit easier to know which files are ours from a glance.